### DISCUSSION TITLE
Comms for final task

### DISCUSSION THREAD

**Discussion by DerryLin-Tamu** [SimDynamX/SpaceTeamsPRO#74] [2024 Lunar SAR Competition Q&A] [answered]
What are the specs for the comms (range specifically)

**Comment by connorjak** [comment, marked as answer]
Antenna height is 4m, rover antenna height is 2.5m with the exception of 2m on the Scout Rovers. Range is infinite but line of sight is required. The elevation data we link to may be useful in planning locations to move comms antennas (beacons) to for good coverage of the search area.
- https://trek.nasa.gov/moon/#v=0.1&x=-6430.13125146415&y=-14629.127643297626&z=9&p=urn%3Aogc%3Adef%3Acrs%3AIAU2000%3A%3A30120&d=&l=Site01_final_adj_5mpp_ColorHillshade%2Ctrue%2C1&l=Site01_final_adj_5mpp_surf%2Ctrue%2C1&l=Site01_final_adj_5mpp_Slope%2Ctrue%2C1&locale=&b=moon&e=-18411.848857054953%2C-24211.157214830815%2C5551.586354126652%2C-5047.098071764438&sfz=&w=
